src/Microsoft.ML.FastTree/Training/OptimizationAlgorithms/OptimizationAlgorithm.cs

```diff
@@ -95,6 +94,7 @@ public ScoreTracker GetScoreTracker(string name, Dataset set, double[] initScore
 
         protected abstract ScoreTracker ConstructScoreTracker(string name, Dataset set, double[] initScores);
 
+        // Regularize a regression tree with smoothing paramter alpha
```

Comment:
>Regularize a regression tree with smoothing paramter alpha [](start = 11, length = 58)

convert to xml comment for intellisense.

Reply:
Sure.

---
In reply to: [251971688](https://github.com/dotnet/machinelearning/pull/2243#discussion_r251971688) [](ancestors = 251971688)